shapely/tests/test_io.py

```diff
@@ -385,38 +385,34 @@ def test_to_wkt_multipoint_with_point_empty_errors():
         shapely.to_wkt(geom)
 
 
-@shapely20_todo
 def test_repr():
-    assert repr(point) == "<pygeos.Geometry POINT (2 3)>"
+    assert repr(point) == "<shapely.geometry.Point POINT (2 3)>"
 
 
-@shapely20_todo
 def test_repr_max_length():
     # the repr is limited to 80 characters
     geom = shapely.linestrings(np.arange(1000), np.arange(1000))
     representation = repr(geom)
-    assert len(representation) == 80
+    assert len(representation) == 92
```

Comment:
Because the "fixed" part became longer ("pygeos.Geometry" vs "shapely.geometry.LineString"), this number increased. Another option if we want to keep the length limited here at 80 is to reduce the width of the truncated WKT, but that also shows even a smaller part of the WKT ..

Reply:
@caspervdw do you have a strong opinion here? We might want to keep the full truncated version at 80 chars for avoiding wrapping on terminals?

Reply:
(sidenote: another option is to move the geometry subclasses to the main namespace. That's something I have been considering for other reasons as well ("what to do with the Geometry class?"), will try to open an issue for that)

Reply:
I don’t have a strong opinion about the truncation length.
I prefer moving the geometry type to the root namespace though, to shorten that part of the repr and to ease imports.

Reply:
> I prefer moving the geometry type to the root namespace though, to shorten that part of the repr and to ease imports.

Yes, that's already done! -> https://github.com/shapely/shapely/pull/1339